picard/util/__init__.py

```diff
@@ -301,7 +301,12 @@ def samefile(path1, path2):
 
 def format_time(ms, display_zero=False):
     """Formats time in milliseconds to a string representation."""
-    ms = float(ms)
+    # Ensure that the input represents a number
+    try:
+        ms = float(ms)
+    except (TypeError, ValueError):
```

Comment:
I don't think hiding the exception is the way to fix this issue.

Reply:
Should we throw the exception instead, and make all callers handle it?

Reply:
How do you reproduce it? Following indications on PICARD-3045 doesn't cause any crash here with current master. Provide the actual trace.

Reply:
Updated the ticket